src/Dommel/DommelMapper.cs

```diff
@@ -2048,7 +2048,7 @@ private sealed class MySqlSqlBuilder : ISqlBuilder
         {
             public string BuildInsert(string tableName, string[] columnNames, string[] paramNames, PropertyInfo keyProperty)
             {
-                return $"insert into {tableName} ({string.Join(", ", columnNames)}) values ({string.Join(", ", paramNames)}); select LAST_INSERT_ID() id";
+                return $"insert into {tableName} ({"`" + string.Join("`, `", columnNames) + "`"}) values ({string.Join(", ", paramNames)}); select LAST_INSERT_ID() id";

```

Comment:
Thanks for the information, I comitted the changes you suggested. I've tested it on my local MySQL Server, and it works as intended.  Since I only have a MySQL Server I don't know how other SQL Servers handle this. Maybe a general fix for all SQL Systems is necessary?